### ISSUE TITLE
Pythons site module could not be loaded

### ISSUE THREAD

**Issue by asolopovas** [vim/vim#717]
Hi, I am using gvim and vim in Windows 10 x64. I spent around 24 hours fiddling with building vim from source but python still does not work in it. Vim from official site (ftp://ftp.vim.org/pub/vim/pc/gvim74.exe) is shutting down each time when I was trying to run python dependent plugin, so after few hours of googling I found the article that lead me to building  x64 version with python x64. 
So I downloaded Visual Studio 2015 and it failed building because of missing Win32.mak. So I downgraded to Visual Studio 2013 and Installed Windows 7 SDK and copied Win32.mak into Win8 sdk include folder. So finally I managed to build it from source. But there is another issue it started, but gives me the following error "E887: Sorry, this command is disabled, the Python's site module could not be loaded". So I tried to build x86 from source and python x86, and it gave me the same error.

**Comment by asolopovas** [comment]
Everything worked with python 2.7.9

**Comment by chrisbra** [comment]
Python 2.7.11 has a nasty bug, that prevents its ussage on Windows. There is an easy workaround however: [link](https://github.com/vim/vim-win32-installer/blob/master/appveyor.bat#L89-L91).

**Comment by voldikss** [comment]
Ｉ've tried python 2.7.13,the latest release when i posted this. It also has this bug, so finally I chose Python 2.7.12

**Comment by zethzeth** [comment]
I had the error on my Mac. I realized that I was running Vim v. 7.4. I uninstalled Vim and installed it again with brew (v. 8.0). When I ran `vim` afterwards, then it was still opening v. 7.4 (I forgot to run it with the `--override-system-vim`-flag. But when I added `alias vim='/usr/local/bin/vim'` in my `.zshrc`-file (the Zshell version of the `.bashrc`- or `.bash_profile`-file), then it worked afterwards.
